### ISSUE TITLE
Opening .R files from file explorer spawns new RStudio instance per file

### ISSUE THREAD

**Issue by SimonDedman** [rstudio/rstudio#7196]
Same behaviour as [this](https://github.com/rstudio/rstudio/issues/6790) but with R files instead.

### System details

    RStudio Edition : Desktop
    RStudio Version : 1.3.1023
    OS Version      : Xubuntu 19.10
    R Version       : 3.6.3

### Steps to reproduce the problem
File explorer is Thunar 1.8.9 (older stable) on XFCE, never had this problem before. Have RStudio open, do some work, doubleclick an R file and it opens in a second instance of RStudio which spawns for it. If I click a third R file (or the same one again), it spawns a third RStudio.

Basically, doubleclicking any R file spawns a new instance of RStudio rather than opening the script in any existing instance.

**Comment by kevinushey** [comment]
Do you see this with Xubuntu 20.04? We generally only support LTS releases of Linux operating systems.

**Comment by ronblum** [comment]
FWIW, I can't reproduce this on Ubuntu 20.04, though I'm using a slightly different configuration
- RStudio Desktop 1.3.1039
- Ubuntu 20.04
- R 3.6.3
- Thunar 1.8.14

**Comment by SimonDedman** [comment]
Hi fellas,

Sadly I can't test in 20.04 because it's bizarrely [incompatible with my machine](https://bugs.launchpad.net/ubuntu/+source/ubuntu-release-upgrader/+bug/1883153) :(

TBH given how unlikely it is that someone else is similarly unable to upgrade, and given Ron's setup is prettymuch vanilla current linux, maybe this isn't worth dev time and I'll just suck it up?

Edit - my bad chaps, it turns out that somehow doubleclicking R files opened them in RStudio yet R files had no default program association in my OS yet. How the OS knew to open them in RStudio without assigning them as such, I don't know. Regardless, manually setting the file/program association has done the trick. Mark this off to 'standard linux weirdness'.

**Comment by ronblum** [comment]
@SimonDedman Thanks for checking, and I appreciate your letting us know! And for what it's worth, TIL about Thunar. :)
